ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_indexing_like_operations.py

```diff
@@ -139,3 +139,33 @@ def test_numpy_put_along_axis(
         axis=axis,
         values=values,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.place",
+    dtype_and_x=helpers.dtype_and_values(
+        num_arrays=3,
+        min_num_dims=1,
+        max_num_dims=3,
+        dtype=["float32", "bool", "float32"],
```

Comment:
Any particular reason for limiting the data dtypes of the arrays to only be `float32`? I feel like thats a bit restrictive. It should ideally take all valid data types unless I'm missing something 🤔

Reply:
You are correct! I fixed that! Thanks